api/src/main/java/org/kuali/rice/kim/service/IdentityManagementService.java

```diff
@@ -1,353 +1,344 @@
-
-    GroupInfo getGroupByName(
-    		@WebParam(name="namespaceCode") String namespaceCode,
-    		@WebParam(name="groupName") String groupName
-    );
-
-    List<String> getParentGroupIds( @WebParam(name="groupId") String groupId);
-    List<String> getDirectParentGroupIds( @WebParam(name="groupId") String groupId);
-
-    @WebMethod(operationName="getGroupIdsForPrincipal")
-    List<String> getGroupIdsForPrincipal( @WebParam(name="principalId") String principalId);
-
-    @WebMethod(operationName="getGroupIdsForPrincipalByNamespace")
-    List<String> getGroupIdsForPrincipal(
-    		@WebParam(name="principalId") String principalId,
-    		@WebParam(name="namespaceCode") String namespaceCode
-    );
-
-    @WebMethod(operationName="getGroupsForPrincipal")
-    List<? extends GroupInfo> getGroupsForPrincipal( @WebParam(name="principalId") String principalId);
-
-    @WebMethod(operationName="getGroupsForPrincipalByNamespace")
-    List<? extends GroupInfo> getGroupsForPrincipal(
-    		@WebParam(name="principalId") String principalId,
-    		@WebParam(name="namespaceCode") String namespaceCode
-    );
-
-    List<String> getMemberGroupIds( @WebParam(name="groupId") String groupId);
-    List<String> getDirectMemberGroupIds( @WebParam(name="groupId") String groupId);
-
-    @WebMethod(operationName="isMemberOfGroup")
-	boolean isMemberOfGroup(
-			@WebParam(name="principalId") String principalId,
-			@WebParam(name="groupId") String groupId
-	);
-
-    @WebMethod(operationName="isMemberOfGroupByNamespace")
-	boolean isMemberOfGroup(
-			@WebParam(name="principalId") String principalId,
-			@WebParam(name="namespaceCode") String namespaceCode,
-			@WebParam(name="groupName") String groupName
-	);
-
-	boolean isGroupMemberOfGroup(
-			@WebParam(name="potentialMemberGroupId") String potentialMemberGroupId,
-			@WebParam(name="potentialParentId") String potentialParentId
-	);
-
-	List<String> getGroupMemberPrincipalIds( @WebParam(name="groupId") String groupId);
-	List<String> getDirectGroupMemberPrincipalIds( @WebParam(name="groupId") String groupId);
-
-    boolean addGroupToGroup(
-    		@WebParam(name="childId") String childId,
-    		@WebParam(name="parentId") String parentId
-    );
-
-    boolean removeGroupFromGroup(
-    		@WebParam(name="childId") String childId,
-    		@WebParam(name="parentId") String parentId
-    );
-
-    boolean addPrincipalToGroup(
-    		@WebParam(name="principalId") String principalId,
-    		@WebParam(name="groupId") String groupId
-    );
-
-    boolean removePrincipalFromGroup(
-    		@WebParam(name="principalId") String principalId,
-    		@WebParam(name="groupId") String groupId
-    );
-
-    GroupInfo createGroup( @WebParam(name="groupInfo") GroupInfo groupInfo);
-    void removeAllGroupMembers( @WebParam(name="groupId") String groupId);
-
-    GroupInfo updateGroup(
-    		@WebParam(name="groupId") String groupId,
-    		@WebParam(name="groupInfo") GroupInfo groupInfo
-    );
-

```

Comment:
The only thing that changed in this class was that I removed the following method to get around the following error:

Caused by: com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 1 counts of IllegalAnnotationExceptions
javax.servlet.http.HttpServletRequest is an interface, and JAXB can't handle interfaces.
	this problem is related to the following location:
		at javax.servlet.http.HttpServletRequest
		at private javax.servlet.http.HttpServletRequest org.kuali.rice.kim.service.jaxws_asm.GetAuthenticatedPrincipalName.arg0
		at org.kuali.rice.kim.service.jaxws_asm.GetAuthenticatedPrincipalName